swedish-oidc-profile.md

```diff
@@ -212,8 +209,66 @@ An OpenID Provider compliant with this profile SHOULD support the PKCE extension
 
 An OpenID Provider MUST NOT allow a Relying Party to use the `plain` code challenge method.
 
+<a name="authentication-request-parameter-extensions"></a>
+### 2.2. Authentication Request Parameter Extensions
+
+This section contains authentication request parameter extensions defined within the scope of this profile.
+
+<a name="client-provided-user-message"></a>
+#### 2.3.1. Client Provided User Message
+
+**Parameter:** `https://claims.oidc.se/1.0/userMessage`
+
+**Description:** When the user message claim is included in an authentication request the issuing client requests
+that the OpenID Provider displays this message to the client in conjunction with the user authentication. 
+
+This profile does not specify how the message should be displayed by the OP, but if the `display` request parameter
+(see 3.1.2.1 of \[[OpenID.Core](#openid-core)\]) is included in the request, and supported by the OP, the provider
+SHOULD display the user message according to the value of the `display` parameter.
+
+**Value type:** The value for the user message request parameter claim is a JSON object<sup>1</sup> 
+with the following fields:
+
+* `message` - A JSON-map where the keys are language tags (\[[RFC5646](#rfc5646)\]) and the map values are
+base64-encodings of the UTF-8 string holding the message to display to the user. The map MUST contain at least
+one language-message pair.
+
+* `mime_type` - The MIME type of the supplied message. This profile defines two possible values that are 
```

Comment:
Do we need to explicitly forbid or allow other values than the two values defined by this profile? E.g. if some OP for some reason would like to support `text/asciidoc` or `text/html`, should that be OK within the scope of the https://claims.oidc.se/1.0/userMessage? I.e. should an OP reject the request or not?

I.e. something akin to sentences such as `Other parameters MAY be sent. ` or `Authorization Servers SHOULD ignore unrecognized request parameters.` commonly seen in the OIDC core spec.

Reply:
Yes. I agree. But a mime type cannot be ignored. As a client I wouldn't appreciate if my request of `text/html` would be interpreted as plain text.

So, we allow for other mime types, but an OP not supporting a mime type must not try to display the message at all.